frontend/kesaseteli/handler/public/locales/fi/common.json

```diff
@@ -14,6 +14,10 @@
     "pageNotFoundLabel": "Sivua ei löytynyt",
     "pageNotFoundContent": "Voi ei, jokin meni nyt valitettavasti vikaan. Kokeile mennä <lnk>tästä</lnk> etusivun kautta takaisin hakemukseen."
   },
+  "403Page": {
+    "forbiddenPageLabel": "Sinulla ei ole oikeuksia",
+    "forbiddenPageContent": "Oikeutesi eivät riitä avaamaan hakemusta. Ota yhteyttä <a>helpdeskiin</a>."
+  },
   "error": {
```

Comment:
Hi, should this also be in en and sv translations..?

Reply:
Handler supports only finnish language at least for now. It might change in the future, that'a why finnish texts are not hard coded to components and we use translation library already.